sphinx_rtd_theme/layout.html

```diff
@@ -9,8 +9,7 @@
 {%- set lang_attr = 'en' if language == None else (language | replace('_', '-')) %}
 
 <!DOCTYPE html>
-<!--[if IE 8]><html class="no-js lt-ie9" lang="{{ lang_attr }}" > <![endif]-->
-<!--[if gt IE 8]><!--> <html class="no-js" lang="{{ lang_attr }}" > <!--<![endif]-->
+<html lang="{{ lang_attr }}" >
```

Comment:
Was modernizer also responsible for the `no-js` class here? I was considering using that in the future for no-JS compatibility on some of the CSS, but that functionality would be easy enough to replicate if so.

Reply:
I think the `no-js` was there by default and then removed by Modernizr.
We can do something similar if we want to support users without JS.